meetings/2021-11-25.md

```diff
@@ -0,0 +1,117 @@
+* SC: Showed "Derived From" relation (`prov:wasDerivedFrom`) in [Notification Protocol](https://solid.github.io/notifications/protocol) as an example to [solid-notifications.md](https://github.com/solid/notifications-panel/blob/3fdc8057d88d6c2e38055557fe62cc3f9c1ba752/proposals/solid-notifications.md) with also link to [Internet Archive snapshot](https://web.archive.org/web/20211014130216/https://github.com/solid/notifications-panel/blob/3fdc8057d88d6c2e38055557fe62cc3f9c1ba752/proposals/solid-notifications.md).
+* eP: You can get the link from the last commit. This link will always exist.
+* RG: Why not delete the file contents and put a message saying it has been moved
+* SC: Normally you don't change the meaning of the resource
+* eP: we can use https://github.com/solid/notifications-panel/blob/3fdc8057d88d6c2e38055557fe62cc3f9c1ba752/proposals/solid-notifications.md which will not disappear.
+* SC: When you delete the file, does that link still work.
+* eP: Yes, that revision is maintained
+* RG: What are we putting in this panel if we remove the MD file?
+* SC: We will keep the use cases and any other proposals that can start from here, plus any highlevel things we can keep here/
+* SC: Once a work item is started, we can say where that work will go.
+* RG: So this is a temporary holiding place for anything related to notifications until we find the right home?
+* SC: It is an incubation space. Not everything will be taken up within the panel.
+
+ACTION: eP to move the webhook document.
+ACTION: SC to delete the notifications protocol under notifications-panel proposals/.
+
+* eP: Do you have a preference for a webhook directory or drop files in the root - I prefer a directory.
+* SC: The protocol was done a particular way which could be used as a pattern. You can see the repo is called notificaitons and if the file is called protocol, e.g., end up with https://solid.github.io/notifications/protocol (using GitHub pages for now for the Editor's Draft). And then if/when a version is published at solid/specification (PR'd), it'll be published under `https://solidproject.org/TR/{shortname}`.
+* eP: Problem with page rendering whilst loading dokieli so suggestion is to use `deferred`.
+* SC: It already loads after a `documentLoaded` event but I will look into it.
+* eP: There is an issue #36 referencing this and loading async.
+* eP: So I will call the file webhooks.md? We can adjust it later if needed.
+* SC: That's good, but it's a description type so should it be webhook-subscription?
+* SC: We should keep the names consistent.
+* eP: OK I'll go with webhook-subscription.md.
+* SC: The problem with the slash directory is that GitHub pages quietly loads index.html but when we move it to a solid server, if spec is on a `/` it will be a problem further down the line. That's why we use an actual name instead of slash. In this case at github.io, `notifications` is the directory and we are not expecting a lot at this root so `protocol` is fine.
```

Comment:
```suggestion
* SC: That's good, but it's a description type so should it be `webhook-subscription`?
* SC: We should keep the names consistent.
* eP: OK I'll go with `webhook-subscription.md`.
* SC: The problem with the slash directory is that GitHub pages quietly load `index.html`, but when we move it to a solid server, if spec is on a `/`, it will be a problem further down the line. That's why we use an actual name instead of slash. In this case at `github.io`, `notifications` is the directory, and we are not expecting a lot at this root, so `protocol` is fine.
```